app/Bot/BotHelper.php

```diff
@@ -132,10 +132,13 @@ public function parseGroupText(Update $update)
 
                 }
             }
-            $this->telegram->deleteMessage([
+
+            $this->telegram->sendMessage([
```

Comment:
Currently, people cannot see who has reported this message. I think it's not the best way to change this, as it will leak the identity of that person. Your thoughts?